### ISSUE TITLE
Tooltip for variants table is in fixed location

### ISSUE THREAD

**Issue by AlistairNWard** [tonydisera/gene.iobio#678]
If you scroll down, the tooltip still always appears above the table, so can be invisible if you scroll far enough, or just chopped off.

**Comment by tonydisera** [comment]
I'm not sure how to fix this.  On way would be to anchor the tooltip so that it doesn't scroll with main window.

**Comment by AlistairNWard** [comment]
Obviously, this is minor. Is it possible to have it scroll with the window, but when the top of the tooltip reaches the top of the screen, it no longer scrolls with the page. I feel like I've seen this behaviour before. BUT, this is hardly top of the list of things to deal with.

**Comment by tonydisera** [comment]
Hi Al.  I'm not sure how to do this.  I'm sure there is a way, but for now, I'm going to create a separate issue.

**Comment by AlistairNWard** [comment]
Yeah, this isn't pressing. Although there is no need to create a new issue. This is already specific to the problem. I'll just throw it back into the backlog.

**Comment by tonydisera** [comment]
I did 'fix' this, but I'm not entirely happy with the outcome.  Now, when hovering over the tooltip, it seems like the tooltip doesn't always show up depending on the vertical scroll position of the window.
